frontend/src/components/ProfileDropdown.vue

```diff
@@ -56,20 +56,22 @@
       </div>
       <div class="border-t border-gray-100"></div>
       <div class="py-1">
-        <router-link to="/setting" class="menu-item" role="menuitem"
-          >Settings</router-link
-        >
+        <router-link to="/setting" class="menu-item" role="menuitem">{{
+          $t("common.settings")
```

Comment:
@LiuJi-Jim I got a question here, our prettierrc set single quote: false. So here the editor would set it to double quote.

However, in place like :title="$t('common.databases')", we have to use single. There is a bit annoying as we need to use slightly different way to write this.

Suggestion?

Reply:
@tianzhou I think that's exactly why vue/recommended prefer single-quote. That helps us to use the same quote style in both JS/TS and <templates>.
I prefer to keep current code in this cycle, then we introduce another auto-fix of the entire frontend codes maybe next week. This will change hundreds of files but entirely automatic which can be trusted as a very safe refactor.